### ISSUE TITLE
''default" keyword in property names

### ISSUE THREAD

**Issue by b12k** [vuejs/vue-loader#263]
```javascript
template syntax error - avoid using JavaScript keyword as property name: "default" in expression :class="{ 'btn-default': channel.id, 'btn-success': !channel.id  }"
```
```javascript
template syntax error - avoid using JavaScript keyword as property name: "default" in expression :class="channel.id ? 'btn-default' : 'btn-success'"
```
How to use Twitter Bootstrap classes in this case?

**Comment by yyx990803** [comment]
I mis-published 9.x as latest on npm - if you are using it with Vue 1.x, switch back to vue-loader@8.x. Otherwise this is a bug in Vue 2.0.

**Comment by b12k** [comment]
Ok. Thanx!
But how to handle this with Vue 2?
Whats wrong with 'btn-default'? Thats a very common thing in CSS frameworks. May be it is better to warn only when 'default' is used as is and not as part of construction?
"Default" is pretty common word for class names.
It also can cause misunderstandings for those who use BEM for CSS.

**Comment by yyx990803** [comment]
should already been fixed in 2.0

vue-loader@9.x will not be compatible with 1.x.
